tests/cpp/aot/aot_save_load_test.cpp

```diff
@@ -146,8 +146,8 @@ TEST(AotSaveLoad, Vulkan) {
   ret_kernel->launch(&host_ctx);
   vulkan_runtime->synchronize();
 
-  // auto x_field = vk_module.get_field("x");
-  // EXPECT_TRUE(x_field);
-  // x_field.copy_to(/*dst=*/x.get());
+  // Retrieve data
+  auto x_field = vk_module->get_field("place");
+  EXPECT_TRUE(x_field);
```

Comment:
```suggestion
  EXPECT_NE(x_field, nullptr);
```

https://google.github.io/googletest/reference/assertions.html